### ISSUE TITLE
Update "Ref a DOM Element" section

### ISSUE THREAD

**Issue by lex111** [the-road-to-learn-react/the-road-to-learn-react#156]
> The examples below have been updated to use the React.createRef() API introduced in React 16.3. If you are using an earlier release of React, **we recommend using callback refs instead**.

From https://reactjs.org/docs/refs-and-the-dom.html

It seems to me it is necessary to mention at least simply about a method with `React.createRef()`?

**Comment by lex111** [comment]
Duplicate of #122?

сс @rwieruch

**Comment by rwieruch** [comment]
Yep. Duplicate. I hadn't the chance to take care of it, because I wasn't able to dive into the new ref API myself :) But it would be a great addition to mention the different variations at least or reference to them more explicit in the exercises. I wouldn't want to bloat the chapter with all different implementations, but it should be clear for the read that there are other possibilities.

**Comment by lex111** [comment]
It would be nice to add this as soon as possible. Just a mention at least.
